cmd/app.go

```diff
@@ -64,13 +52,6 @@ func (app *App) PrepareConfig() error {
 
 	app.Config.Services.Clean()
 
-	if app.Config.Settings.SkipShuffle {
-		log.Infof("Skip shuffeling service order.")
-	} else {
-		log.Infof("Shuffling service list")
-		app.Config.Services.Shuffle()
-	}
```

Comment:
We don't need it, since I also removed bulk deployments.

Also, I currently see no purpose of creating a full stack reBuy service infrastructure out of nowhere, at the moment. But we might have to talk about this.